operator/controllers/resources/suite_test.go

```diff
@@ -0,0 +1,180 @@
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resources
+
+import (
+	"context"
+	"math/rand"
+	"path/filepath"
+	"testing"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/gravitational/teleport/lib/service"
+	"github.com/stretchr/testify/require"
+	core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	ctrl "sigs.k8s.io/controller-runtime"
+	kclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/envtest"
+
+	"github.com/gravitational/teleport/api/client"
+	"github.com/gravitational/teleport/api/types"
+	"github.com/gravitational/teleport/integration"
+	"github.com/gravitational/teleport/integration/helpers"
+	resourcesv2 "github.com/gravitational/teleport/operator/apis/resources/v2"
+	resourcesv5 "github.com/gravitational/teleport/operator/apis/resources/v5"
+	//+kubebuilder:scaffold:imports
+)
+
+func fastEventually(t *testing.T, condition func() bool) {
+	require.Eventually(t, condition, time.Second, 100*time.Millisecond)
+}
+
+func clientForTeleport(t *testing.T, teleportServer *helpers.TeleInstance, userName string) *client.Client {
+	identityFilePath := integration.MustCreateUserIdentityFile(t, teleportServer, userName, time.Hour)
+
+	teleportClient, err := client.New(context.Background(), client.Config{
+		Addrs:       []string{teleportServer.GetAuthAddr()},
+		Credentials: []client.Credentials{client.LoadIdentityFile(identityFilePath)},
+	})
+	require.NoError(t, err)
+
+	return teleportClient
+}
+
+func defaultTeleportServiceConfig(t *testing.T) (*helpers.TeleInstance, string) {
+	teleportServer := helpers.NewInstance(helpers.InstanceConfig{
+		ClusterName: "root.example.com",
+		HostID:      uuid.New().String(),
+		NodeName:    integration.Loopback,
+	})
+
+	rcConf := service.MakeDefaultConfig()
+	rcConf.DataDir = t.TempDir()
+	rcConf.Auth.Enabled = true
+	rcConf.Proxy.Enabled = true
+	rcConf.Proxy.DisableWebInterface = true
+	rcConf.SSH.Enabled = true
+	rcConf.Version = "v2"
+
+	roleName := validRandomResourceName("role-")
+	unrestricted := []string{"list", "create", "read", "update", "delete"}
+	role, err := types.NewRole(roleName, types.RoleSpecV5{
+		Allow: types.RoleConditions{
+			Rules: []types.Rule{
+				types.NewRule("role", unrestricted),
+				types.NewRule("user", unrestricted),
+			},
+		},
+	})
+	require.NoError(t, err)
+
+	operatorName := validRandomResourceName("operator-")
+	_ = teleportServer.AddUserWithRole(operatorName, role)
+
+	err = teleportServer.CreateEx(t, nil, rcConf)
+	require.NoError(t, err)
+
+	return teleportServer, operatorName
+}
+
+func startKubernetesOperator(t *testing.T, teleportClient *client.Client) kclient.Client {
+	testEnv := &envtest.Environment{
+		CRDDirectoryPaths:     []string{filepath.Join("..", "..", "config", "crd", "bases")},
```

Comment:
Changing the CRD output path from `operator/config/crd/bases/` to `examples/chart/teleport-cluster/charts/teleport-operator/templates` means this will stop working

We have to either write to the two places or change this path to the subchart location
I prefer the latter